app/controllers/file_manager_controller.rb

```diff
@@ -34,8 +34,8 @@ def index
       if File.size(absolute_path) > 1_000_000 || params[:download]
         send_file absolute_path
       else
-        mime_type = MimeMagic.by_path(absolute_path).type
-        if mime_type.split('/').first == 'text'
+        mime_type = MimeMagic.by_path(absolute_path)
+        if mime_type.nil? || mime_type.type.split('/').first == 'text'
```

Comment:
This might not be the best approach because it would attempt to show binary files too (assuming there are no extensions)

How about doing it similar to submissions controller?

```ruby
  def is_binary_file?(file)
    mm = MimeMagic.by_magic(file)
    mm.present? && (!mm.text? && (mm.subtype != "pdf"))
  end
```

You can drop the pdf check, and display anything that isn't a binary file.

Reply:
okay, fixed.